php/class-pwawp-app-manifest.php

```diff
@@ -0,0 +1,171 @@
+<?php
+/**
+ * PWAWP_APP_Manifest class.
+ *
+ * @package PWA
+ */
+
+/**
+ * PWAWP_APP_Manifest class.
+ *
+ * Mainly copied from Jetpack_PWA_Manifest and Jetpack_PWA_Helpers.
+ */
+class PWAWP_APP_Manifest {
+
+	/**
+	 * The query arg that is present in a request for the app manifest.
+	 *
+	 * @var string
+	 */
+	const MANIFEST_QUERY_ARG = 'pwawp_manifest';
+
+	/**
+	 * The theme color to use if no dynamic values are present.
+	 *
+	 * @var string
+	 */
+	const FALLBACK_THEME_COLOR = '#fff';
+
+	/**
+	 * The default manifest icon sizes.
+	 *
+	 * Copied from Jetpack_PWA_Helpers::get_default_manifest_icon_sizes().
+	 * Based on a conversation in https://github.com/GoogleChrome/lighthouse/issues/291
+	 *
+	 * @var int[]
+	 */
+	public $default_manifest_icon_sizes = array( 192, 512 );
+
+	/**
+	 * Inits the class.
+	 *
+	 * Mainly copied from Jetpack_PWA_Manifest::__construct().
+	 */
+	public function init() {
+		add_action( 'wp_head', array( $this, 'manifest_link_and_meta' ) );
+		add_action( 'amp_post_template_head', array( $this, 'manifest_link_and_meta' ) );
```

Comment:
This AMP action should be omitted. It should be up to the AMP plugin to add the link.

Reply:
Thanks, [a7414](https://github.com/xwp/pwa-wp/pull/13/commits/a74140d231fc31958c4010429abd2637dfbb08b9#diff-f875394ddc0965b517a89f44c0e877e2L46) removes the AMP action.